### PR TITLE
Migrate `delete_message` function to Helix

### PR DESCRIPTION
**Pull request by ALazyMeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

I noted that the api supports the deletion of all messages, so I've separated out the functions in case someone wants to use the delete_all_messages function

### CODE REVIEW

**Comment by ALazyMeme** [comment]
Not tested yet

**Comment by ALazyMeme** [comment]
Works perfectly
image

File: pajbot/apiwrappers/twitch/helix.py [hunk 1]
```diff
@@ -615,3 +615,28 @@ def send_chat_announcement(self, channel_id: str, bot_id: str, message: str, aut
             authorization=authorization,
             json={"message": message},
         )
+
+    def delete_chat_messages(
```
Comment (ALazyMeme) [inline pajbot/apiwrappers/twitch/helix.py:619]:
```suggestion
    def _delete_chat_messages(
```

File: pajbot/apiwrappers/twitch/helix.py [hunk 2]
```diff
@@ -615,3 +615,28 @@ def send_chat_announcement(self, channel_id: str, bot_id: str, message: str, aut
...
+            {"broadcaster_id": channel_id, "moderator_id": bot_id, "message_id": message_id},
+            authorization=authorization,
+        )
+
+    def delete_single_message(self, channel_id: str, bot_id: str, authorization, message_id: str) -> None:
+        """Deletes a single message from the chatroom using the message_id.
+        channel_id, bot_id and message_id are all required fields. bot_id must match the user ID
+        in authorization.
+        An exception is raised if there are any invalid or missing details."""
+        self.delete_chat_messages(channel_id, bot_id, authorization, message_id)
```
Comment (ALazyMeme) [inline pajbot/apiwrappers/twitch/helix.py:636]:
```suggestion
        self._delete_chat_messages(channel_id, bot_id, authorization, message_id)
```

File: pajbot/apiwrappers/twitch/helix.py [hunk 3]
```diff
@@ -615,3 +615,28 @@ def send_chat_announcement(self, channel_id: str, bot_id: str, message: str, aut
...
+
+    def delete_all_messages(self, channel_id: str, bot_id: str, authorization) -> None:
+        """Deletes all messages from the chatroom.
+        channel_id and bot_id are required fields. bot_id must match the user ID in authorization.
+        An exception is raised if there are any invalid or missing details."""
+        self.delete_chat_messages(channel_id, bot_id, authorization)
```
Comment (ALazyMeme) [inline pajbot/apiwrappers/twitch/helix.py:642]:
```suggestion
        self._delete_chat_messages(channel_id, bot_id, authorization)
```

**Comment by ALazyMeme** [comment]
I couldn't find an easier way to edit this on mobile

**Comment by ALazyMeme** [comment]
The `delete_all_messages` function also works using `!eval bot.twitch_helix_api.delete_all_messages(bot.streamer.id, bot.bot_user.id, bot.bot_token_manager)`:
image

**Review by pajlada — APPROVED** [review]

**Pull request merged**